oz/core/actions.py

```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```

Comment:
That's not what it does though. You can check the source code (there's a link in the documentation).